tests/integrations/python/tests/test_azure.py

```
@@ -0,0 +1,1744 @@
+
+        error = exc_info.value
+        # Should get an error for invalid model
+
+        # Test with unsupported file format (if applicable)
+        with pytest.raises(Exception) as exc_info:
+            azure_client.audio.transcriptions.create(
+                model=get_model("azure", "transcription"),
+                file=("test.txt", b"text file content", "text/plain"),
+            )
+
+        error = exc_info.value
```

Ruff (0.15.0)

[error] Local variable `error` is assigned to but never used

Remove assignment to unused variable `error`

(F841)